src/site/antora/modules/ROOT/nav.adoc

```diff
@@ -52,14 +52,21 @@
 ** xref:manual/systemproperties.adoc[]
 * xref:manual/usage.adoc[]
 * xref:manual/cloud.adoc[]
-* xref:manual/async.adoc[]
-* xref:manual/garbagefree.adoc[]
+* xref:manual/lookups.adoc[]
+* xref:manual/appenders.adoc[]
+* xref:manual/layouts.adoc[]
+** xref:manual/json-template-layout.adoc[]
+* xref:manual/filters.adoc[]
 * xref:manual/extending.adoc[]
-** xref::manual/customconfig.adoc[]
+** xref:manual/customconfig.adoc[]
 * xref:manual/plugins.adoc[]
+* xref:manual/customconfig.adoc[]
+* xref:manual/customloglevels.adoc[]
 * xref:manual/jmx.adoc[]
 * xref:manual/logsep.adoc[]
 * xref:manual/performance.adoc[]
+** xref:manual/async.adoc[]
+** xref:manual/garbagefree.adoc[]
```

Comment:
I find this kind of hierarchy a little bit weird.

I would expect to see a short text under "Performance" and 3 children: "Log4j API usage", "Asynchronous loggers" and "Garbage Free".

Reply:
As can be seen in `performance.adoc`, it is not just about Log4j API, async. loggers, and garbage-free. There are more concerns that need to be covered. I think `performance.adoc` is in a good state now:

* It is a _complete_ portal to all performance-related knobs
* The content is ordered from generic to specific

My rationale for keeping async. loggers and garbage-free in separate pages is:

* They are optional and advanced features
* Their setup spans across both API and implementation (unfit for the easy-to-follow procedural checklist established in `performance.adoc`)
* Their content is massive
* Link compatibility concerns